### PR TITLE
[New Post]: Levelling up Your Releases: Reduce Risk with Blue/Green Deployments

### PR DESCRIPTION
**Pull request by wirjo**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Review by cherbk — APPROVED** [review]

**Comment by scribwell** [comment]
@crichey  This deep dive is ready for your technical review.

**Comment by scribwell** [comment]
@wirjo @setheliot @cobusbernard  I scanned the article and it's not a deep dive, it's a blog post so I tagged it as such and flagged it for one-step review.

File: posts/levelling-up-your-releases-a-deep-dive-into-blue-green-deployments/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - aws
+  - resilience
+  - codedeploy
+  - deployment
+  - devops
+  - ci-cd
+  - ecs
+  - api-gateway
+  - serverless
+authorGithubAlias: wirjo
```
Comment (setheliot) [inline posts/levelling-up-your-releases-a-deep-dive-into-blue-green-deployments/index.md:14]: please add to resilience wave 😁

File: posts/levelling-up-your-releases-a-deep-dive-into-blue-green-deployments/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+### Using the routing layer of your architecture
+
+While we recommend starting with fully managed capabilities outlined, you can also consider implementing blue/green deployment at the routing layer of your architecture by [using Amazon Route 53](https://docs.aws.amazon.com/whitepapers/latest/blue-green-deployments/update-dns-routing-with-amazon-route-53.html) or [swapping the Auto Scaling Group behind an Elastic Load Balancer](https://docs.aws.amazon.com/whitepapers/latest/blue-green-deployments/swap-the-auto-scaling-group-behind-elastic-load-balancer.html). This can provide you with greater control on the process.
+
+![Blue/green by swapping the Auto Scaling Group behind an Elastic Load Balancer](images/swap-auto-scaling-group.png)
+
+While there are many ways to implement, the goal should be to automate the process, and ultimately reduce errors and costs. Overall, you should choose the appropriate implementation based on your architecture, the nature of your application, and goals for your organisation.
+
+## A Feal-World Experience
```
Comment (setheliot) [inline posts/levelling-up-your-releases-a-deep-dive-into-blue-green-deployments/index.md:77]: "Real"

**Review by setheliot — APPROVED** [review]
See one typo

Otherwise, very high-quality content -- happy to see this published, and feel free to add to resilience wave

**Pull request merged**